app/src/main/java/org/mozilla/fenix/components/toolbar/ToolbarUIView.kt

```diff
@@ -173,6 +172,8 @@ class ToolbarUIView(
     }
 
     companion object {
+        private const val PROGRESS_BOTTOM = 0
```

Comment:
Does 0 / 1 map to the Gravity constants?

Reply:
Yep!

Reply:
AC customs gravity progress constants